srtcore/core.cpp

```diff
@@ -11594,6 +11826,7 @@ int CUDTGroup::sendBroadcast(const char* buf, int len, SRT_MSGCTRL& w_mc)
     // Now we can go to the idle links and attempt to send the payload
     // also over them.
 
+    // { sendBroadcast_ActivateIdlers
```

Comment:
Commented code

Reply:
Same. I didn't do the refax on this function, but places for further refax are identified here. Some of the smaller parts (usually with `send_` prefix) are the same for Broadcast and Backup, so in the future one version for both can be provided.